recipes/libneuroml/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "libNeuroML" %}
+{% set version = "0.2.43" %}
+{% set sha256 = "8128415feef1f601c1b33866b2eb43afb43447e8be22233fac82300e8e5b7537" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python
+    - lxml
+    - pytables
+    - jsonpickle
+    - simplejson
+    - pymongo
+    - cython
```

Comment:
given that your package depends on cython at runtime, you probably want to add the `{{ compiler('c') }}` `{{ compiler('cxx') }}` dependencies in here

Reply:
@mariusvniekerk : Thanks for having a look at the recipe. I'm checking back with the original authors why they listed Cython as a dependency in the first place, it is actually very likely that we don't need it here. That said, from experience I can say that adding the  `{{ compiler(...)}}` syntax as a runtime dependency does not work on `conda-forge` -- it will install the `toolchain` package which is only useful on the CI servers and will not work on the user's system (see: https://conda-forge.org/docs/meta.html#compilers).